src/views/Article/index.vue

```diff
@@ -453,6 +460,48 @@ export default {
       await this.getArticlesList(this.signId, 1);
       this.refreshing = false;
     },
+    // 删除文章
+    delArticleButton() {
+      if (this.article.author !== this.currentUsername) {
```

Comment:
这种情况下，不应该显示删除按钮。

Reply:
```suggestion
 <Button v-if="isMe" @click="delArticleButton" ..........
 
     isMe() {
      console.log('isme', this.article, this.currentUsername);
      return this.article.author === this.currentUsername;
    },
    
```

其实按钮上面已经做了隐藏，重复这个判断只是预防万一按钮显示了（还能挡一次）

Reply:
cool